packages/integrations/image/README.md

```diff
@@ -63,11 +63,205 @@ export default {
+
+The desired aspect ratio of the output image. Combine with either `width` or `height` to automatically calculate and crop the other dimension.
+
+A `string` can be provided in the form of `{width}:{height}`, ex: `16:9` or `3:4`.
+
+A `number` can also be provided, useful when the aspect ratio is calculated at build time. This can be an inline number such as `1.777` or inlined as a JSX expression like `aspectRatio={16/9}`.
+
+### `<Picture /`>
+
+#### src
+
+<p>
+
+**Type:** `string` | `ImageMetadata` | `Promise<ImageMetadata>`<br>
+**Required:** `true`
+</p>
+
+Source for the original image file.
+
+For images in your project's repository, use the `src` relative to the `public` directory. For remote images, provide the full URL.
+
+#### alt
+
+<p>
+
+**Type:** `string`<br>
+**Default:** `undefined`
+</p>
+
+If provided, the `alt` string will be included on the built `<img />` element.
+
+#### sizes
+
+<p>
+
+**Type:** `string`<br>
+**Required:** `true`
+</p>
+
+The HTMLImageElement property `sizes` allows you to specify the layout width of the image for each of a list of media conditions.
+
+See [MDN](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/sizes) for more details.
+
+#### widths
+
+<p>
+
+**Type:** `number[]`<br>
+**Reuqired:** `true`
+</p>
+
+The list of sizes that should be built for responsive images. This is combined with `aspectRatio` to calculate the final dimensions of each built image.
+
+```astro
+// Builds three images: 400x400, 800x800, and 1200x1200
+<Picture src={...} widths={[400, 800, 1200]} aspectRatio="1:1" />
+```
+
+#### aspectRatio
+
+<p>
+
+**Type:** `number` | `string`<br>
+**Required:** `true`
+</p>
+
+The desired aspect ratio of the output image. This is combined with `widhts` to calculate the final dimensions of each built image.
+
+A `string` can be provided in the form of `{width}:{height}`, ex: `16:9` or `3:4`.
+
+A `number` can also be provided, useful when the aspect ratio is calculated at build time. This can be an inline number such as `1.777` or inlined as a JSX expression like `aspectRatio={16/9}`.
+
+#### formats
+
+<p>
+
+**Type:** Array<'avif' | 'jpeg' | 'png' | 'webp'><br>
+**Default:** `undefined`
+</p>
+
+The output formats to be used in the optimized image. If not provided, `webp` and `avif` will be used in addition to the original image format.
+
+### `getImage`
+
+This is the helper function used by the `<Image />` component to build `<img />` attributes for the transformed image. This helper can be used directly for more complex use cases that aren't currently supported by the `<Image />` component.
+
+This helper takes in an object with the same properties as the `<Image />` component and returns an object with attributes that should be included on the final `<img />` element.
+
+This can helpful if you need to add preload links to a page's `<head>`.
+
+```astro
+---
+import { getImage } from '@astrojs/image';
+
+const { src } = await getImage('../assets/hero.png');
+---
```

Comment:
This does not work for me. Maybe I configured something incorrectly, but I tried to follow the README as it was stated here. Any idea why this happens? (same for `getPicture`)
```
'getImage' is not exported by node_modules/@astrojs/image/components/index.ts, imported by src/components/Picture.astro
file: /Users/fidge123/dev/private/project/src/components/Picture.astro:1:3
1: ---
      ^
2: import { getImage } from '@astrojs/image';
 error   'getImage' is not exported by node_modules/@astrojs/image/components/index.ts, imported by src/components/Picture.astro
  File:
    /Users/fidge123/dev/private/project/src/components/Picture.astro
  Code:
    1: ---
          ^
    2: import { getImage } from '@astrojs/image';

error Command failed with exit code 1.
```

Reply:
`getImage` and `getPicture` aren't currently public, this PR will make them available in the next release 👍 

> The responsive image section should point out how to use this to support higher resolution screens (retina) which look better with 2x/3x scaling.

Definitely agree we can build up some really nice content for best practices here!  I've actually been leaning towards writing a few blog posts that dives deeper into responsive images and how to use these components, the README could link to those for full details

Reply:
Thank you for the clarification! I'm looking forward to those blog posts 😊